src/lib/logging.ts

```diff
@@ -11,14 +11,17 @@ LevelToNum.set("warning", 2)
 LevelToNum.set("info", 3)
 LevelToNum.set("debug", 4)
 
+// I'm sure I shouldn't have to generate this
+const dummy = new Config.default_config()
```

Comment:
Anyone know enough about Typescript to tell me how to avoid this? I'm sure I'm missing something obvious

Reply:
I tried for ~1 hour and I don't think it's possible.